### PR TITLE
allow data uris as images

### PR DESCRIPTION
**Pull request by mmalerba**

### CODE REVIEW

File: firebase.json [hunk 1]
```diff
@@ -45,7 +45,7 @@
           {
             "key": "Content-Security-Policy",
             // Keep in sync with `angular.json`.
```
Comment (mmalerba) [inline firebase.json:47]: I'm not sure what this comment means, I don't see this being set in angular.json

**Review by crisbeto — APPROVED** [review]

File: firebase.json (hunk 1 above)
Comment (devversion) [reply firebase.json:47]: It's this one: https://github.com/angular/material.angular.io/blob/main/angular.json#L114

**Review by devversion — APPROVED** [review]
LGTM but we should update `angular.json` for similar dev behavior

File: firebase.json (hunk 1 above)
Comment (mmalerba) [reply firebase.json:47]: Huh, don't know why I couldn't find that

**Pull request merged**